processing/src/test/java/io/druid/query/groupby/epinephelinae/BufferGrouperTest.java

```diff
@@ -117,6 +127,34 @@ public void testGrowing()
   }
 
   @Test
+  public void testGrowing2()
```

Comment:
@gianm these two tests increased testing time quite significantly (https://github.com/druid-io/druid/issues/4402#issue-235728060). Can we remove one of these tests maybe testGrowing2() because they looks similar?